rust/src/lib/net_state.rs

```diff
@@ -212,6 +214,14 @@ impl NetworkState {
         self
     }
 
+    /// When set to true, the network state be applied will not merging current
```

Comment:
The sentence is not well constructed and it's difficult to understand. Can you reword it?

Reply:
Removed. New approach been used.